apps/domain/lib/domain/network.ex

```diff
@@ -4,7 +4,7 @@ defmodule Domain.Network do
 
   @cidrs %{
     ipv4: %Postgrex.INET{address: {100, 64, 0, 0}, netmask: 10},
-    ipv6: %Postgrex.INET{address: {64768, 0, 0, 0, 0, 0, 0, 0}, netmask: 106}
+    ipv6: %Postgrex.INET{address: {64_768, 0, 0, 0, 0, 0, 0, 0}, netmask: 106}
```

Comment:
Suggested update to use Firezone-designated IPv6 prefix of [`fd00:0222:2011:1111/64`](https://github.com/firezone/connlib/blob/master/apple/Sources/Connlib/CallbackHandler.swift#L40):

```suggestion
    ipv6: %Postgrex.INET{address: {64_768, 546, 8_209, 4_369, 0, 0, 0, 0}, netmask: 64}
```

Why the change?

Unfortunately it's almost a guarantee we'll hit IP collisions on devices in large orgs (IPv4 CGNAT especially), so we'll need to be thoughtful about picking a range that reduces the chance of collision while being easy for admins to write exceptions / workarounds for.
* The RFC-designated private IPv6 subnets are in the `fc00:/7` range, and so many administrators will base their `/64` subnets at `fc00::0` or `fd00::0` or `fe00` etc. Offsetting to `fd00:0222:2011:1111` reduces the chance for collision somewhat since it's more obscure.
* `/64`s are the very commonly used for private IPv6 subnets; admins will be more familiar dealing with `/64`s than `/106s`.
* Some device OSes [don't handle uncommon IPv6 subnets well](https://github.com/WireGuard/wireguard-apple/blob/2fec12a6e1f6e3460b6ee483aa00ad29cddadab1/Sources/WireGuardKit/PacketTunnelSettingsGenerator.swift#L138)

It will be harder later to expand this range from a `/106` to a `/64`, so perhaps it's better to pick the larger range now. In the future we could even disable IPv4 for a tenant and use IPv6-only for Firezone addresses (this might happen due to IPv4 CGNAT conflict).

Reply:
cc @conectado and @rbhitchcock if they have any experience/input with regarding our chosen IPv6 prefix.

Reply:
I think [this is the section](https://datatracker.ietf.org/doc/html/rfc4193#section-3.2.2) of the relevant RFC on generating unique IPV6 (they recommend /48 for prefix and 16 bits for subnetting)

We can use an address generated from [this site](https://unique-local-ipv6.com) that follows (almost) that spec to minimize collision.

Reply:
@conectado Yeah, I considered using a random one, but decided we could make it easier to remember using an easter egg like `2021:1111` which is the date Firezone, Inc. was incorporated. Open to other suggestions ;-).

That would give us a prefix of `fd00:2021:1111::/48`, but we probably also want to select the `/16` right?

Reply:
We don't need a really big IPv6 pool because we still rely mostly on IPv4 and not likely will be allocating more than 4 million IPs per account (they are scoped). 4m should be enough. As for the offset I trust you to pick it :).

Reply:
@jamilbk the problem using that date is that there is basically no entropy increasing the chances of collision but it might be worth having something easy to remember.

For selecting the /16 we can go with `::`.

Reply:
@AndrewDryga K, sounds good. How about we go with `fd00:2021:1111::/106` here? We document publicly that Firezone's unique local prefix is `fd00:2021:1111::/48` and then reserve the remaining upper portion of our `/48` for later use.

```suggestion
    ipv6: %Postgrex.INET{address: {64_768, 8_209, 4_369, 0, 0, 0, 0, 0}, netmask: 106}
```